instat/dlgOneWayFrequencies.vb

```diff
@@ -21,8 +21,7 @@ Public Class dlgOneWayFrequencies
     Private bReset As Boolean = True
     Private bResetSubdialog As Boolean = False
     Private clsSjTab As New RFunction
-    Private clsSjPlot, clsPlotGrid, clsSelect As New RFunction
-    Private clsTableBaseOperator, clsGraphBaseOperator As New ROperator
+    Private clsSjPlot, clsPlotGrid As New RFunction

```

Comment:
It's not just that it does the same thing. If we use the pipes here it's inconsistent with how we do this for all other dialogs. So we should only use it if we have to.